.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: ci
+
+on:
+  push:
+    branches: [main, actions-test]
+  pull_request:
+    branches: [main, actions-test]
+
+jobs:
+  build_and_test:
+    name: build and test (Elixir ${{matrix.elixir}} | Erlang/OTP ${{matrix.otp}})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - elixir: 1.12.x
+            otp: 24.x
+          - elixir: 1.13.x
+            otp: 24.x
+          - elixir: 1.14.x
+            otp: 24.x
+          - elixir: 1.13.x
+            otp: 25.x
+          - elixir: 1.14.x
+            otp: 25.x
+    env:
+      MIX_ENV: test
+    steps:
+      - uses: actions/checkout@v3
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{matrix.otp}}
+          elixir-version: ${{matrix.elixir}}
+
+      - name: Install Dependencies
+        run: |
+          mix local.hex --force
+          mix local.rebar --force
```

Comment:
i don't think these are necessary

Reply:
Huum, I'm not sure. But I removed anyways